AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml

```diff
@@ -1,2 +1,2 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<project name="AcumaticaSurveyPkg" level="0" description="" />
\ No newline at end of file
+<project name="Customization" level="0" description="" />
```

Comment:
@markoan we need to keep project name as AcumaticaSurveyPkg

Reply:
Got it. It got changed when I exported the content and wanted to keep the same folder (it saves to the folder with the name you use).
Are you renaming this manually each time you export the module?